File: src/main/java/com/kyle/guess/GameFor.java
package com.kyle.guess;

import java.util.Random;
import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {
        int scr = new Random().nextInt(10)+1; //bound的數字代表0~9，()外+1 代表0~9變成1~10
        Scanner sca =new Scanner(System.in);
        System.out.println(scr);
        for(int i=1;i<=4;i++){
            System.out.println("plz enter the number("+i+"/4):");
            int num =sca.nextInt();
            if(num>scr){
                System.out.println("lower");
            }else if(num<scr){
                System.out.println("higher");
            }else{
                System.out.println("that's right! the number is "+scr);
            break;
            }
        }

    }
}

File: src/main/kotlin/com/kyle/GameFor.kt
package com.kyle

import java.util.*

fun main(args:Array<String>) {
    val scr =Random().nextInt(10)+1
    println(scr)
    var sca =Scanner(System.`in`)
    for (i in 1..4) {
        println("plz enter the number (${i}/4): ")
        var num = sca.nextInt()
        if (num > scr) {
            println("lower")
        } else if (num < scr) {
            println("higher")
        } else{
        println("That;s right the number is $scr")
        break
        }
    }

}